### PR TITLE
HW0 + HW1 (edit mw3071.md to add project contributions and ideas, added README description and md files for projects with links to README)

### PR DESCRIPTION
**Pull request by mw3071**
HW0B Submission

### CODE REVIEW

File: mw3071.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h1>Open Source Development</h1>
+  
+Name: Matthew Wang
+
+UNI: mw3071
+
+GitHub: [link](https://github.com/mw3071)
+
+GitHub README: [link](https://github.com/mw3071/project-proposals-s2023/blob/main/README.md)
```
Comment (timkpaine) [inline mw3071.md:9]: this is for HW1

File: mw3071.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<h2>Project Proposal</h2>
+TBD
```
Comment (timkpaine) [inline mw3071.md:12]: at least 1

File: mw3071.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+<h2>Project Contributions</h2>
+
+TBD
```
Comment (timkpaine) [inline mw3071.md:16]: at least 1

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Review by mw3071 — COMMENTED** [review]
Added project proposals and contribution ideas. Also added HW1 github README

File: mw3071.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+GitHub README: [link](https://github.com/mw3071/mw3071/blob/main/README.md)
+
+<h2>Project Proposal</h2>
+Discord bot to show what computers are open at the Wallach Gaming Lounge
+Discord bot that allows users to match with other students in the same class so they can study together
+
+<h2>Project Contributions</h2>
+
+Discord.py/discord.js
```
Comment (timkpaine) [inline mw3071.md:17]: add markdown file with project description

File: columbia-discord-bot.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (timkpaine) [inline columbia-discord-bot.md:1]: move to `projects/python` or whatever language you're using

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by timkpaine** [comment]
Looks like you're switching this to be HW1, which means you don't have a HW0 submission

**Comment by mw3071** [comment]
Since HW1 is an add on to HW0, how do I submit both fixes at the same time?

**Pull request closed without merging**